src/_posts/2020-12-07-bootstrap-5-beta-1.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title: Bootstrap 5 Beta 1
+video: y2bVIBwpCTA
+date: 2020-12-07 11:00:00
```

Comment:
The time might not be right but I see we are doing this in other posts too. The default timezone is the build machine's timezone. We can change it in the config, but maybe it's not worth messing with it since some permalinks might change.

Another approach would be to specify the timezone in the frontmatter. I haven't tried it, but it might also work.

Anyway, that's for another PR, I just noticed it here :)